src/components/MainGame/PlayerInfoPanel.vue

```diff
@@ -52,6 +52,9 @@
             <button class="btn btn-primary btn-lg" v-on:click="discardSelected" style="border-radius: 40px">
               Discard
             </button>
+            <button class="btn btn-lg btn-warning" v-on:click="reDraw" style="border-radius: 40px; margin-top: 5px;">
```

Comment:
I like the solution, it appears as if it works. However, the UI solution could use some work. For now could you lay the buttons horizontally? I think the divs for the buttons will have to be extracted and have the width extended separately from the parent div right now, but I'm not sure. 

The colour of the button also doesn't fit with the current game theme. The bootstrap 'info button' colour may be a better fit. 

Hopefully that makes sense.